src/main/kotlin/no/nav/familie/ef/sak/ekstern/EksternBehandlingController.kt

```diff
@@ -46,13 +59,51 @@ class EksternBehandlingController(private val pdlClient: PdlClient,
      */
     @PostMapping("finnes/flere-identer")
     @ProtectedWithClaims(issuer = "azuread", claimMap = ["roles=access_as_application"])
-    fun finnesBehandlingForPersonIdenter(@RequestParam("type") stønadstype: Stønadstype?,
-                                         @RequestBody personidenter: Set<String>): Ressurs<Boolean> {
+    fun finnesBehandlingForPersonIdenter(
+        @RequestParam("type") stønadstype: Stønadstype?,
+        @RequestBody personidenter: Set<String>
+    ): Ressurs<Boolean> {
         return finnesBehandlingFor(personidenter, stønadstype)
     }
 
-    private fun finnesBehandlingFor(personidenter: Set<String>,
-                                    stønadstype: Stønadstype?): Ressurs<Boolean> {
+    @GetMapping("hent/{personidenter}")
+    @ProtectedWithClaims(issuer = "azuread", claimMap = ["roles=access_as_application"])
+    fun erBehandlingerUtdaterteForPersonidenter(
+        @PathVariable personidenter: Set<String>
+    ): Ressurs<Boolean> {
+        if (personidenter.isEmpty()) {
+            return Ressurs.failure("Finner ikke identer til personen")
+        }
+        if (personidenter.any { it.length != 11 }) {
+            return Ressurs.failure("Støtter kun identer av typen fnr/dnr")
+        }
+
+        val behandlinger = hentAlleBehandlingIDer(personidenter)
+        if (behandlinger.isEmpty()) {
+            return Ressurs.failure("Det finnes ingen behandlinger for oppgitte personidenter")
+        }
+        return erBehandlingerUtdaterte(behandlinger)
+    }
+
+    private fun hentAlleBehandlingIDer(personidenter: Set<String>): Set<UUID> {
+        var behandlinger = mutableSetOf<UUID>()
+        Stønadstype.values().forEach {
+            behandlingRepository.finnSisteBehandlingSomIkkeErBlankett(it, personidenter)?.let { behandlinger.add(it.id) }
+        }
+        return behandlinger
+    }
+
+    private fun erBehandlingerUtdaterte(behandlingIDer: Set<UUID>): Ressurs<Boolean> {
+        var tilkjenteYtelser = mutableSetOf<TilkjentYtelse>()
+        behandlingIDer.forEach { tilkjenteYtelser.add(tilkjentYtelseService.hentForBehandling(it)) }
+        val senesteTomDatoAvAndeler : LocalDate = tilkjenteYtelser.maxOf { it.andelerTilkjentYtelse.maxOf { it.stønadTom } }
+        return Ressurs.success(senesteTomDatoAvAndeler < LocalDate.now().minusYears(1))
+    }
```

Comment:
For meg virker det som att denne returnerer true hvis man har en stønad som er eldre enn 12 måneder, er det ønskelig? 

Burde man kanskje kunne gjøre det uten å ha e "temp set"
```kotlin
private fun harStønadSiste12Måneder(behandlingIDer: Set<UUID>): Boolean {
        val sisteStønadsdato = behandlingIDer
                                .map(tilkjentYtelseService::hentForBehandling)
                                .mapNotNull { it.andelerTilkjentYtelse.maxOfOrNull(AndelTilkjentYtelse::stønadTom) }
                                .maxOfOrNull { it } ?: LocalDate.MIN
        return sisteStønadsdato > LocalDate.now().minusYears(1)
    }
```